scripts/copy_env_feature_flag_settings.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Description:
+#   Copies feature flag settings from a Kubernetes ConfigMap to a local Docker env.dev file.
+#   Currently, only supports env vars that end in _IS_ENABLED. Some feature flag settings rely on env vars that do not follow this pattern.
+#   The script appends the settings to the existing env.dev file by default.
+#
+# Prerequisites:
+#   - The K8s repository must be in the same directory as ea_enterprise
+#   - Git must be installed and configured
+#
+# Directory Structure:
+#   local_development/
+#   ├── ea_enterprise/
+#   │   └── env.dev
+#   └── <k8s_repo_dir_name>/
+#       └── environments/
+#           └── <env_name>/
+#               └── config/
+#                   └── env-configmap.yaml
+#
+# Usage:
+#   ./copy_env_feature_flag_settings.sh <k8s_repo_dir_name> <env_name>
+#
+# Arguments:
+#   k8s_repo_dir_name    Name of the K8s repository directory
+#   env_name             Name of the environment to copy settings from
+#
+# Example:
+#   ./copy_env_feature_flag_settings.sh cme_k8s pvt-3
+#
+# Note:
+#   Existing feature flags in env.dev will not be removed or deduplicated
+
+set -e  # Exit on any error
+
+# Validate input parameters
+if [ "$#" -ne 2 ]; then
+    echo "Error: Required parameters missing"
+    echo "Usage: $0 <k8s_repo_dir_name> <env_name>"
+    exit 1
+fi
+
+# Validate K8S repository name
+if [ "$1" != "cme_k8s" ] && [ "$1" != "dchbx_k8s" ]; then
+    echo "Error: Invalid K8s repository name. Must be either 'cme_k8s' or 'dchbx_k8s'"
+    exit 1
+fi
+
+# Verify directories exist
+if [ ! -d "../../$1" ]; then
+    echo "Error: K8s repository directory '../../$1' not found"
+    exit 1
+fi
+
+if [ ! -d "../../$1/environments/$2" ]; then
+    echo "Error: Environment '$2' not found in repository"
+    exit 1
+fi
+
+K8S_REPO="$1"
+ENV_NAME="$2"
+K8S_PATH="../../$K8S_REPO"
+ENTERPRISE_PATH="../ea_enterprise"
+CONFIG_MAP_PATH="./environments/$ENV_NAME/config/env-configmap.yaml"
+TEMP_FILE="feature_flags.txt"
```

Comment:
i only recently learned about `mktemp`, if that seems like something you'd want to use here.